config/routes.rb

```diff
@@ -54,6 +54,13 @@
     end
   end
 
+  # used for payments_controller_spec.rb
+  resources :payments do
+    member do
+      get :redirect_to_authorize
+    end
+  end
+
```

Comment:
You shouldn't need to add a route here. It should be possible to write a request spec using the existing route.

`rails routes` tells me:

```
authorize_payment GET      /payments/:id/authorize(.:format)          payments#redirect_to_authorize
```